test/GitHub.InlineReviews.UnitTests/ViewModels/NewInlineCommentThreadViewModelTests.cs

```diff
@@ -1,165 +1,165 @@
-            target.Comments[0].CommitEdit.Execute();
-
-            session.Received(1).PostReviewComment(
-                "New Comment",
-                "COMMIT_SHA",
-                "file.cs",
-                Arg.Any<IReadOnlyList<DiffChunk>>(),
-                5);
-        }
-
-        [Test]
-        public void AddsCommentToCorrectDeletedLine()
-        {
-            var session = CreateSession();
-            var file = CreateFile();
-
-            file.Diff.Returns(new[]
-            {
-                new DiffChunk
-                {
-                    Lines =
-                    {
-                        new DiffLine { OldLineNumber = 17, DiffLineNumber = 7 }
-                    }
-                }
-            });
-
-            var target = new NewInlineCommentThreadViewModel(
-                Substitute.For<ICommentService>(),
-                session, file, 16, true);
-
-            target.Comments[0].Body = "New Comment";
-            target.Comments[0].CommitEdit.Execute();
-
-            session.Received(1).PostReviewComment(
-                "New Comment",
-                "COMMIT_SHA",
-                "file.cs",
-                Arg.Any<IReadOnlyList<DiffChunk>>(),
-                7);
-        }
-
-        IPullRequestSessionFile CreateFile()
-        {
-            var result = Substitute.For<IPullRequestSessionFile>();
-            result.CommitSha.Returns("COMMIT_SHA");
-            result.Diff.Returns(new[]
-            {
-                new DiffChunk
-                {
-                    Lines =
-                    {
-                        new DiffLine { NewLineNumber = 11, DiffLineNumber = 5 }
-                    }
-                }
-            });
-            result.RelativePath.Returns("file.cs");
-            return result;
-        }
-
-        IPullRequestSession CreateSession()
-        {
-            var result = Substitute.For<IPullRequestSession>();
-            result.RepositoryOwner.Returns("owner");
-            result.LocalRepository.Name.Returns("repo");
-            result.LocalRepository.Owner.Returns("shouldnt-be-used");
-            result.PullRequest.Returns(new PullRequestDetailModel { Number = 47 });
-            result.User.Returns(new ActorModel());
-            return result;
-        }
-
-        void RaisePropertyChanged<T>(T o, string propertyName)
-            where T : INotifyPropertyChanged
-        {
-            o.PropertyChanged += Raise.Event<PropertyChangedEventHandler>(new PropertyChangedEventArgs(propertyName));
-        }
-    }
-}
+﻿////using System.Collections.Generic;
+////using System.ComponentModel;
+////using GitHub.InlineReviews.Services;
+////using GitHub.InlineReviews.ViewModels;
+////using GitHub.Models;
+////using GitHub.Services;
+////using GitHub.ViewModels;
+////using NSubstitute;
+////using NUnit.Framework;
+
+////namespace GitHub.InlineReviews.UnitTests.ViewModels
+////{
+////    public class NewInlineCommentThreadViewModelTests
```

Comment:
What is up with these tests? 🙂

Reply:
Ah yes! I removed that class. Need to delete this file.